Gems/StartingPointCamera/Code/Source/StartingPointCamera/StartingPointCameraUtilities.cpp

```diff
@@ -26,38 +26,27 @@ namespace Camera
         return "";
     }
 
-    //////////////////////////////////////////////////////////////////////////
-    /// This methods will 0 out a vector component and re-normalize it
-    //////////////////////////////////////////////////////////////////////////
-    void MaskComponentFromNormalizedVector(AZ::Vector3& v, VectorComponentType vectorComponentType)
+    void MaskComponentFromNormalizedVector(AZ::Vector3& v, bool ignoreX, bool ignoreY, bool ignoreZ)
```

Comment:
question: Before it looked like it was only possible to ignore one axis at a time, but now we can ignore all 3 at the same time? What happens if you do this? Will this 'freeze' the camera?

Reply:
Right now it will throw a warning that you are trying to normalize a zero vector and return that zero vector. Maybe it could be a good idea to always have at least one checkbox activated.

Reply:
This is probably okay for now, in a perfect would it would be nice to avoid it, but in practice it's likely not a huge issue (so long as nothing bad happens, I take it the camera will then just not move?)